### ISSUE TITLE
Web3 Grants Foundation Research

### ISSUE THREAD

**Issue by Hodlatoor** [ubiquity/business-development#6]

**Comment by sergfeldman** [comment]
As I never did this before, I created a draft PR intended to check everything before the real grant application. 
This draft PR is not planned as a real application. 

@pavlovcik may I ask whether you checked 
1) Our wallet 
2) Software license (I put Apache-2.0)
3) Name of the team leader (it is kind of doxxed that you dislike)

---

The change requests from W3F

1) https://github.com/w3f/Grants-Program/pull/1718#pullrequestreview-1412461538

2) https://github.com/w3f/Grants-Program/pull/1718#issuecomment-1534214914

---

After all the changes we should create a new PR according to all the requirements, ie
- The initial PR contains only one commit (squash and force-push if needed)
The other requirements are defined https://github.com/w3f/Grants-Program#pencil-process

Also, Alvaro should monitor the Ubiquity email address because he is determined a contact person

**Comment by sergfeldman** [comment]
Steps if we want to complete and submit this grant application:

- Define the software license. As I understand by the comments in this issue, this is for Jon.

- Update the content: add the table with deliverables according to W3F comments in the draft PR, change Alex name according to his input. I can do this.

- Create new PR with 1 commit. Alex mentioned in the comments in this issue that he wanted to do this.

**Comment by 0x4007** [comment]
> Define the software license. As I understand by the comments in this issue, this is for Jon.

MIT

> Create new PR with 1 commit. Alex mentioned in the comments in this issue that he wanted to do this.

I can yes. Just tag me when you want me to.

**Comment by sergfeldman** [comment]
> > Define the software license. As I understand by the comments in this issue, this is for Jon.
> 
> MIT
> 
> > Create new PR with 1 commit. Alex mentioned in the comments in this issue that he wanted to do this.
> 
> I can yes. Just tag me when you want me to.

@pavlovcik Thank you!

**Comment by Hodlatoor** [comment]
ok so this is good to submit correct?

**Comment by 0x4007** [comment]
@sergfeldman please post the link to the source content (on notion I guess) and the repository that I should open a pull request to when it is confirmed to be completed. Thanks!

**Comment by sergfeldman** [comment]
@pavlovcik @Hodlatoor In the comments to the draft PR, W3F representative requested to fill in the tables with deliverables. 
The main points of these tables are related to Substrate modules, Substrate chain, libraries, etc.

I have added the tables to our application, but for many questions, I put N/A because we do not plan to provide them.
Also, I put the MIT license and updated Alex's name according to his input.

The current version of the application is here
https://github.com/sergfeldman/Grants-Program/blob/DevPool_by_UbiquityDAO/applications/DevPool_by_UbiquityDAO.md

Please, let me know if I should change something.

**Comment by 0x4007** [comment]
> The current version of the application is here
https://github.com/sergfeldman/Grants-Program/blob/DevPool_by_UbiquityDAO/applications/DevPool_by_UbiquityDAO.md

You should add all documents in this repository because 
1. we can't edit the contents of your personal repositories.
2. we can find all the relevant documents in one place.

I will let you know what needs to be changed by changing the document directly. 

For educational purposes I will let you post the document on this repository and then we can go from there.

**Comment by sergfeldman** [comment]
According to the W3F Grant program, it is required to fork their repository and create 1 new file with our application
https://github.com/w3f/Grants-Program#pencil-process

To apply for the grant, it is only needed to paste the content to this file and create the PR to the W3F repository according to all the requirements, ie
- The initial PR contains only one commit (squash and force-push if needed)

---

As requested by @pavlovcik I tried to add all documents to this repository.
I have not figured out how to fork the W3F repository here. 

However, I saved the content of the grant application file in various ways in this repository:

- Uploaded file to the comment
[W3F grant application.md](https://github.com/ubiquity/partnerships/files/11669556/W3F.grant.application.md)

- Created PR 
https://github.com/ubiquity/partnerships/pull/16

- Created discussion 
https://github.com/ubiquity/partnerships/discussions/17

I`m ready to continue the activity for educational purposes.
